kngsild/src/main/kotlin/io/egm/kngsild/model/ApplicationError.kt

```diff
@@ -1,9 +1,10 @@
 package io.egm.kngsild.model
 
-sealed class ApplicationError(open val message: String)
+sealed class ApplicationError(override val message: String) : Exception(message)
```

Comment:
I'm a bit concerned by the fact that all application errors are now exceptions. This is not the way to do it in FP.

I think we should not try to use an application error when it is not possible to build an `AuthUtils` object. Instead, throw an `IllegalArgumentException` like it is done for instance with a Kotlin require call (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require.html).